test/jasmine-test/spec/coreTests.js

```diff
@@ -969,13 +969,20 @@ var coreTests = [
         message: "should escapeJs a string with double quotes when using the j filter"
       },
       {
-        name:     "JSON.stringify filter",
+        name:     "escapeJSON filter",
         source:   "{obj|js|s}",
         context:  { obj: { id: 1, name: "bob", occupation: "construction" } },
         expected: JSON.stringify({ id: 1, name: "bob", occupation: "construction" }),
         message: "should stringify a JSON literal when using the js filter"
       },
       {
+        name:     "escapeJSON filter with bad characters",
+        source:   "{obj|js|s}",
+        context:  { obj: { name: "<<\u2028testLS \u2029testPS"} },
+        expected: '{"name":"&lt;&lt;\\u2028testLS \\u2029testPS"}',
```

Comment:
This seems non-ideal. Now there's no way for me to get angle bracket literals when using `|js`, even if I explicitly say I want to turn off HTML escaping.